src/components/Equity/Equity.tsx

```diff
@@ -57,6 +57,11 @@ export const Equity = ({ assetId, accountId }: EquityProps) => {
   const totalEquityBalance = useAppSelector(state => selectEquityTotalBalance(state, filter))
   const equityRows = useAppSelector(state => selectAssetEquityItemsByFilter(state, filter))
 
+  const shouldDisplayEquityRows = useMemo(
+    () => equityRows.length > 0 || isLoading,
+    [equityRows.length, isLoading],
+  )
+
```

Comment:
Not strictly related to this PR but wondering if we want to tackle l.82 while at it i.e `isLoading` instead of `portfolioLoading` currently

Reply:
Yeah agree, used it here too